File: Challenge/Varuni-Punchihewa.dart
// https://github.com/Varuni-Punchihewa

void main() {
  for (int i = 1; i <= 100; i++) {
    if (i % 3 == 0 && i % 5 == 0) {
      print('Spider');
    } else if (i % 3 == 0) {
      print('Rat');
    } else if (i % 5 == 0) {
      print('Ghost');
    } else {
      print(i);
    }
  }
}

File: Challenge/Varuni-Punchihewa.cpp
// https://github.com/Varuni-Punchihewa

#include <iostream>
using namespace std;

int main()
{
    for (int i = 1; i <= 100; i++)
    {

        if (i % 3 == 0 && i % 5 == 0)
        {
            cout << "Spider" << endl;
        }
        else if (i % 3 == 0)
        {
            cout << "Rat" << endl;
        }
        else if (i % 5 == 0)
        {
            cout << "Ghost" << endl;
        }
        else
        {
            cout << i << endl;
        }
    }

    return 0;
}